### PR TITLE
Add doc

### PR DESCRIPTION
**Pull request by christophmluscher**
add doc string for functions

### CODE REVIEW

File: meta/system.py [hunk 1]
```diff
@@ -487,6 +603,16 @@ def align(self, name, corpus, flow, feature_scorer, **kwargs):
     def estimate_mixtures(
         self, name, corpus, flow, old_mixtures=None, alignment=None, prefix="", **kwargs
     ):
+        """
+        :param str name:
+        :param str corpus:
+        :param str flow:
+        :param str|list[str]|tuple[str]|rasr.FlagDependentFlowAttribute old_mixtures:
```
Comment (michelwi) [inline meta/system.py:610]: Sure that `rasr.FlagDependentFlowAttribute` describes a mixture file?

File: meta/system.py [hunk 2]
```diff
@@ -647,6 +807,19 @@ def recog_and_optimize(
         prefix="",
         **kwargs,
     ):
+        """
+        :param str name:
+        :param str corpus:
+        :param str|list[str]|tuple[str]|rasr.FlagDependentFlowAttribute flow:
+        :param str|list[str]|tuple[str]|rasr.FeatureScorer feature_scorer:
+        :param float pronunciation_scale:
+        :param  floatlm_scale:
```
Comment (michelwi) [inline meta/system.py:816]: typo
```suggestion
        :param float lm_scale:
```

File: meta/system.py [hunk 3]
```diff
@@ -727,6 +912,18 @@ def create_nn_feature_scorer(
         prior_file=None,
         **kwargs,
     ):
+        """
+        :param str name:
+        :param str corpus:
+        :param int|types.FunctionType feature_dimension:
+        :param int|types.FunctionType output_dimension:
+        :param str|list[str]|tuple[str]|rasr.FlagDependentFlowAttribute prior_mixtures:
```
Comment (michelwi) [inline meta/system.py:920]: is `FlagDependentFlowAttribute` correct for mixture file?

**Review by michelwi — CHANGES_REQUESTED** [review]

File: meta/system.py [hunk 4]
```diff
@@ -647,6 +807,19 @@ def recog_and_optimize(
...
     ):
+        """
+        :param str name:
+        :param str corpus:
+        :param str|list[str]|tuple[str]|rasr.FlagDependentFlowAttribute flow:
+        :param str|list[str]|tuple[str]|rasr.FeatureScorer feature_scorer:
+        :param float pronunciation_scale:
+        :param  floatlm_scale:
+        :param bool parallelize_conversion:
+        :param dictlattice_to_ctm_kwargs:
```
Comment (michelwi) [inline meta/system.py:818]: typo
```suggestion
        :param dict lattice_to_ctm_kwargs:
```

File: meta/system.py (hunk 1 above)
Comment (christophmluscher) [reply meta/system.py]: You are right. It is not. It should be Sisyphus Path object

**Review by michelwi — APPROVED** [review]

**Review by JackTemaki — APPROVED** [review]
I am not sure I would find errors even if there are, so this is just an approve from me...

**Pull request merged**